config/collectors.js

```diff
@@ -1,9 +1,12 @@
 const Discord = require('discord.js');
 const fs = require('fs');
 
-function collectCommands(client, commandsDir) {
+function collectCommands({client, regularCommandsDir, slashCommandsDir}) {
```

Comment:
I'm curious what the reasoning is for changing this function to accept an object instead of individual params

Reply:
As the number of parameters grew to 3 structuring it in an object with associated object destructuring was done to simplify the logic and make it more clear what the object properties were for vs using 1st, 2nd, 3rd argument. See: https://github.com/ryanmcdermott/clean-code-javascript#functions